### ISSUE TITLE
People interchangably use symbols and strings for package names in specs, which can be a problem on GemStone

### ISSUE THREAD

**Issue by pdebruic** [dalehenrich/metacello-work#382]
In GemStone #'ConfigurationOfSeaside3' does not equal 'ConfigurationOfSeaside3'

In the GemStone version of Metacello can we get a asString send in there when it is looking up the package name in the list of packages in the repo (which are strings)

I've been manually fixing configs as I find them.

**Comment by pdebruic** [comment]
And it might be a tool like versioneer thats using Symbols for the package names.  Dont know.

**Comment by dalehenrich** [comment]
Yeah this is a persistent problem that causes trouble all over the 
place, since in Pharo is there is virtually now useful reason for 
differentiating between Symbols and Strings. Consequently they get used 
interchangeably all over the place and no one ever notices ...

Dale
